src/BlockTypes/CollectionPriceFilter.php

```diff
@@ -48,16 +48,10 @@ protected function render( $attributes, $content, $block ) {
 			'maxPrice'          => $max_price,
 			'minRange'          => $min_range,
 			'maxRange'          => $max_range,
-			'formattedMinPrice' => $formatted_min_price,
```

Comment:
I saw no reason for us to do this here. This state will be calculated directly from other state here, so afaict there is no reason to do this.